src/Raven.Server/Documents/Queries/AST/Query.cs

```diff
@@ -28,6 +28,8 @@ public sealed class Query
         public ValueExpression FilterLimit;
 
         public List<(QueryExpression Expression, OrderByFieldType FieldType, bool Ascending)> CachedOrderBy;
+        
+        public HashSet<StringSegment> Parameters = new HashSet<StringSegment>(StringSegmentComparer.OrdinalIgnoreCase);
```

Comment:
at the end we agreed that parameters are case sensitive

Reply:
Still pending